src/Backend/Modules/FormBuilder/Layout/Templates/Edit.html.twig

```diff
@@ -280,7 +281,7 @@
                       <li class="checkbox">
                         <label for="textboxReplyTo" class="control-label">
                           {% form_field textbox_reply_to %} {{ 'lbl.ReplyTo'|trans|ucfirst }}
-                          <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpReplyTo'|trans }}"></abbr>
+                          <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpReplyTo'|trans }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320